script/setup/install-runc

```diff
@@ -16,13 +16,14 @@
 
 #
 # Builds and installs runc to /usr/local/go/bin based off
-# the commit defined in vendor.conf
+# the commit defined in go.mod
```

Comment:
I think we no longer need to sync runc binary version with vendored runc version.

Reply:
I prefer doing this in another PR.